tsa-test/src/test/kotlin/org/ton/examples/checkers/CheckersTest.kt

```diff
@@ -146,6 +153,50 @@ class CheckersTest {
         )
     }
 
+    @Ignore("Transactions rollback is not supported")
+    @Test
+    fun transactionRollBackTest() {
+        val sender = extractResource(transactionRollBackTestData.sender)
+        val receiver = extractResource(transactionRollBackTestData.receiver)
+        val senderContract = getFuncContract(
+            sender,
+            FIFT_STDLIB_RESOURCE,
+            isTSAChecker = true
+        )
+        val receiverContract = getFuncContract(receiver, FIFT_STDLIB_RESOURCE)
+
+
+        val communicationScheme = emptyMap<ContractId, TvmContractHandlers>()
+
+        val options = TvmOptions(
+            intercontractOptions = IntercontractOptions(
+                communicationScheme = communicationScheme,
```

Comment:
There is no inter-contract communication here, why do you specify inter-contract scheme?